src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -71,17 +71,15 @@ ucs_status_t uct_ib_mlx5_completion_with_err(uct_ib_iface_t *iface,
                                              uct_ib_mlx5_txwq_t *txwq,
                                              ucs_log_level_t log_level)
 {
-    ucs_status_t  status         = UCS_ERR_IO_ERROR;
-    char          err_info[256]  = {};
-    char          wqe_info[256]  = {};
-    char          peer_info[128] = {};
-    union ibv_gid dgid           = {};
-    uint16_t      wqe_index;
-    uint32_t      qp_num;
-    void          *wqe;
-    unsigned      rqn;
-    uint16_t      dlid;
-    int           is_global;
+    ucs_status_t       status         = UCS_ERR_IO_ERROR;
+    char               err_info[256]  = {};
+    char               wqe_info[256]  = {};
+    char               peer_info[128] = {};
+    uint16_t           wqe_index;
+    uint32_t           qp_num;
+    void               *wqe;
+    unsigned           rqn;
+    struct ibv_ah_attr ah_attr;
```

Comment:
formatting seems inconsistent with other code, should not align var names (only =)